hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml

```diff
@@ -182,6 +182,14 @@
     <name>yarn.resourcemanager.principal</name>
   </property>
 
+  <property>
+    <description>
+      The configuration key for enabling or disabling the auto-correction of container allocation.
```

Comment:
Would you document more details?
- what the auto correction means
- when YARN administrator should enable or disable

I'm +1 if it's addressed.

Reply:
ack.